x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -34,19 +39,23 @@ export const pkgToPkgKey = ({ name, version }: { name: string; version: string }
 export async function fetchList(params?: SearchParams): Promise<RegistrySearchResults> {
   const registryUrl = getRegistryUrl();
   const url = new URL(`${registryUrl}/search`);
-  if (params && params.category) {
-    url.searchParams.set('category', params.category);
+  if (params) {
+    if (params.category) {
+      url.searchParams.set('category', params.category);
+    }
+    if (params.experimental) {
+      url.searchParams.set('experimental', params.experimental.toString());
+    }
   }
 
   return fetchUrl(url.toString()).then(JSON.parse);
 }
 
-export async function fetchFindLatestPackage(
-  packageName: string,
-  internal: boolean = true
-): Promise<RegistrySearchResult> {
+export async function fetchFindLatestPackage(packageName: string): Promise<RegistrySearchResult> {
```

Comment:
from what I saw in the code, the current usage of `fetchFindLatestPackage` implies finding the latest version given a package name _regardless_ of if it's an internal package or experimental one. both internal and experimental always needs to be true to enable searching across _all_ registry packages, so that's why I removed the existing `internal` param and didn't replace it with `options`